joomla/joomla-cms
# [3.10] Privacy Consent wording I agree and No

Pull Request for Issue #22198 .

### Summary of Changes
see #22198 


Testing Instructions


Actual result BEFORE applying this Pull Request

![Screenshot (2031)](https://user-images.githubusercontent.com/54195688/156433447-c0b0d154-927c-4520-87b2-a8c7290c4a24.png)



Expected result AFTER applying this Pull Request
'No' is replaced by 'I do not agree'
